app/controllers/concerns/organizational.rb

```diff
@@ -12,7 +12,16 @@ def current_organization
     @current_organization ||= current_user&.casa_org
   end
 
+  def current_role
```

Comment:
@jmkoni, @compwron, @ds29 I've added this `current_role` here to handle displaying the user's role on the sidebar based on the currently logged in user type. With that said, I'm now considering creating two different `_sidebar` partials, one for the `:user` auth scope, and another for `:all_casa_admin`, and render the correct sidebar based on the current authenticated scope. This would allow us to:

1. Keep the sidebar a little leaner by avoiding quite a few conditional rendering logic that we currently need;
2. Better isolate functionality pertaining to `:user` vs `:all_casa_admin` authentication scopes.

Any concerns with that approach?

Reply:
> Any concerns with that approach?

@drborges I like the suggestion. Separating the concerns would reduce the conditionals and simplify the view to be primarily comprised of the relevant navigational items.

Were you thinking to do this as part of this PR or a separate issue? I would suggest a separate issue given that this PR solves the current issue.

Reply:
Sounds like a plan 👍🏼